src/test/resources/features/actions/line_mover.feature

```diff
@@ -62,36 +89,213 @@ Feature: Line mover
       | checkpoint  | input          | ""      | # comment |
```

Comment:
Again what is the purpose of variables `comment` and `content` here?

Reply:
Done